lib.sh

```diff
@@ -300,7 +300,7 @@ common_finalize() {
     $SUDO rm -rf /var/lib/cloud/instanc*
     $SUDO rm -rf /root/.ssh/*
     $SUDO rm -rf /etc/ssh/*key*
-    $SUDO rm -rf /tmp/*
+    $SUDO rm -rf /tmp/* /var/tmp/automation_images
```

Comment:
Maybe consider not doing this?  Could be handy for someone doing archaeology on an old release-branch CI VM wondering how it was built.

Reply:
I interpret "not doing this" as "undoing the added /var/tmp". So done.